src/report/QUHtmlReport.cpp

```diff
@@ -226,7 +226,7 @@ void QUHtmlReport::appendLyrics(QDomNode &parent) {
 			div.setAttribute("id", QVariant(rn).toString());
 
 		QDomElement h1 = _report.createElement("h1");
-		h1.appendChild(_report.createTextNode(QString("%3. %1 - %2").arg(song->artist()).arg(song->title()).arg(rn)));
+		h1.appendChild(_report.createTextNode(QString("%3. %1 - %2").arg(song->artist(), song->title(), QString::number(rn))));

```

Comment:
For multi-args, there seems to be no overload. So for a single .arg(), it can contain an int, but for multi-arg, it seems all args need to be QStrings (`no matching member function for call to 'arg'`).